docs/source/topics/packaging.rst

```diff
@@ -35,9 +35,9 @@ write yourself.
 There are two options for handling python package dependencies:
 
 * **requirements.txt** - During the packaging process, chalice will
-  run ``pip install -r requirements.txt`` in a virtual environment
-  and automatically install 3rd party python packages into the deployment
-  package.
+  install any packages it find or build compatible wheels for. Specifically
```

Comment:
I think the wording is off here.  Did you mean "...install any packages it finds or builds compatible wheels for" maybe?